tests/main/uc20-create-partitions-reinstall/task.yaml

```
@@ -1,5 +1,15 @@
 summary: Integration tests for the bootstrap.Run autodetect
 
+details: |
+    When working on a core system, snapd contains logic to install the system
+    from scratch, creating partitions and file systems as necessary. The test
+    uses snapd internal API loaded through a helper Go program, to exercise that
+    in a hybrid unit/integration test, and observe the outcome.
+
+    This test focuses on the re-installation scenario. A similar test looks at
+    the first installation scenario and yet another test looks at creating
+    encrypted file systems.
```

I'd drop reference to other tests, nobody will remember to update it if said tests are changed/removed
```suggestion
    This test focuses on the re-installation scenario.
```

---

I added it specifically so that peopel would look and see if a change makes sense elsewhere.